modules/admin-ui/src/main/resources/public/org/opencastproject/adminui/languages/lang-en_US.json

```diff
@@ -427,7 +427,10 @@
            "ACL": "Access policy"
          },
          "SUMMARY": {
-           "CAPTION": "Summary"
+           "CAPTION": "Summary",
```

Comment:
Bit late for changing the translations innit?  I don't think this can go into 5.x as is...

Reply:
Otherwise tests look good.  My sole complaint is this translation change.

Reply:
@KatrinIhler It is indeed true that a proposal was accepted that forbids translation changes into release branches. Can you factor out the translation change and do that in a separate PR against develop? You could just do it without i18n support here - at the end of the day, it is too late to re-translate 5.x anyway...

Reply:
You're both right, and I had a feeling you would complain about that. ;) Just wanted to know what you would suggest... I've decided to simply leave out the 'Input'-prefix in the summary completely instead of having it be untranslated here since that's pretty ugly and obvious in my opinion and it's still understandable what these table entries mean without it, and I'm gonna reintroduce it in a separated PR targeted at develop.